plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -158,12 +160,20 @@ private static JSONObject generateRootConfiguratorObject() {
         return rootConfiguratorObject;
     }
 
-    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute) {
+    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute, Object configObject) {
         if (attribute.type.getName().equals("java.lang.String")) {
             attributeSchema.put(attribute.getName(),
                 new JSONObject()
                     .put("type", "string"));
         } else {
+            ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
+            try {
+
+                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(Jenkins.getInstance().getDescriptor().getClass()
+                    , attribute.getName()));
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
If you want to record an error, it should go to system Logger at least

Reply:
Yeah this is still in its debugging stage @oleg-nenashev